casa/Json/test/tJsonKVMap.in

```diff
@@ -0,0 +1,20 @@
+# This defines the input for a file.
```

Comment:
this is not valid json, it can't have comments and the number formatting is much stricter
this can be ok for the input, but not in the output (e.g. .write() as a comment argument)